.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches: [ master, add/gh-action-prod ]
 
 name: Deploy Production CGK
```

Comment:
- Ensure the format of the commit details output is clear and consistent for easier readability.